include/rocksdb/file_system.h

```diff
@@ -825,6 +852,20 @@ class FSRandomAccessFile {
     return IOStatus::NotSupported("InvalidateCache not supported.");
   }
 
+  //  Asynchronous Read of the data requested in FSReadRequest. This is a
+  //  asynchronous call, i.e it will return after submitting the request.
+  //  When read request is completed callback function specified in cb will be
+  //  called with result populated in FSReadResponse. This API also populates
+  //  IOHandle. req contains the request offset and size, but the result and
```

Comment:
Clarify that the callee should populate IOHandle with a unique value. Also, replace "will" with "should" in the description